src/GuessControlOld.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
```

Comment:
These `xxxOld` versions of the components were meant to be just for reference while you were working, so they're not something intended to be checked in. Please delete these two files and commit/push them before merging!